lecture_4/03_bet_acceptance_api/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/koines/code-cadets-2021/lecture_4/03_bet_acceptance_api
+
+go 1.16
+
+require (
+	github.com/gin-gonic/gin v1.7.2
+	github.com/kelseyhightower/envconfig v1.4.0
+	github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
```

Comment:
Genuinely interested - how did you find this library? First thing I find when googling "golang generate UUID" is [this](https://github.com/google/uuid). And that library seems better in any way I look at:
* It's still actively maintained
* Has lots more users
* It's using go modules so you can actually reference an actual version instead of a commit hash

Reply:
We used this library in the third lecture.